### ISSUE TITLE
 [Interop layer] test json schema validation

### ISSUE THREAD

**Issue by jamestranovich-noaa** [weather-gov/weather.gov#1848]
### Description

We have a few PHP tests that do very similar things and I believe they can all be combined in one test suite, since they all build a block (in PHP parlance) and verify the JSON schema against the output. 

- [ ] verify JSON against `alertInfo.json` schema: `AlertInfo.php.test`
- [ ] verify JSON against `daily.json` schema: `DailyForecast.php.test`
- [ ] verify JSON against `alerts.json` schema: `Alerts.php.test`
- [ ] verify JSON against `hourly.json` schema: `HourlyForecast.php.test`

**Comment by colinmurphy01** [comment]
@jamestranovich-noaa @greg-does-weather does this need to be included in this sprint in order to deploy to staging?

**Comment by jamestranovich-noaa** [comment]
No. I would consider this low priority and "nice to have" as we don't actually do anything meaningful with JSON validation (yet). The only reason to include this is to make sure we have test parity, but honestly I don't think this should be a blocker to interop deployment. If we get this in soon, then great! Otherwise I think we can wait. (Feel free to disagree @greg-does-weather )

**Comment by greg-does-weather** [comment]
Agreed, the schema tests are a way to ensure we know when the data structure changes so nothing accidentally changes, and nothing changes without being noted in the tests.

Instead of validating against the structures from the PHP blocks, maybe we should validate it against the JSONschema doc? I think the complexity with this one will be mocking all the API and database calls. I think we should be able to mock them using bundled data from the proxy, or alternatively (and maybe preferably?) we let this particular test actually make network calls. But in any case, I agree with James's assessment.

**Comment by colinmurphy01** [comment]
Grouping with the other non-blocked interop issues in the `Now` priority and off the `backlog`

**Comment by colinmurphy01** [comment]
Is this `done`?

**Comment by colinmurphy01** [comment]
Moving this back into backlog, unless @jamestranovich-noaa you planned on picking this up this sprint?
